library/build.gradle

```diff
@@ -31,6 +32,7 @@ android {
 
 dependencies {
     api "com.android.support:support-annotations:$supportLibVersion"
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
should use `implementation` instead of compile

Reply:
done